bin/link_check.py

```diff
@@ -165,11 +165,17 @@ def test_links(self, new_url, urls):
                          'status': -1,
                          'resources': new_url[url]}
                 else:
-                    res={'timestamp': now,
-                         'status':response.status_code}
-                    if response.status_code != requests.codes.ok:
-                        res['resources'] = new_url[url]
-                        res['org'] = urls.get(url, None)
+                    try:
+                        res={'timestamp': now,
+                             'status':response.status_code}
+                        if response.status_code != requests.codes.ok:
+                            res['resources'] = new_url[url]
+                            res['org'] = urls.get(url, None)
+                    except:
```

Comment:
it's not an Exception subclass? Then I'd really like to know what it is!  We don't use bare `except`s (unless you're also using a bare `raise`) because they catch things they shouldn't.